### ISSUE TITLE
EXC_BAD_ACCESS (code=1, address=0x18) in operationQueue

### ISSUE THREAD

**Issue by kurko** [wvteijlingen/Spine#68]
Since I upgraded to the latest master (the big rewrite), I'm getting the following error in `Spine.swift`:

```
EXC_BAD_ACCESS (code=1, address=0x18) in operationQueue
```

### When

* It **doesn't** happen when I run the tests individually. The integration tests in my project pass.
* It happens when I run the entire test suite.

I don't reset Spine between tests, but it seems that something inside Spine one operation is leaking into another. I really don't know how to debug this...

### The error

<img width="525" alt="spine-error" src="https://cloud.githubusercontent.com/assets/70312/13081598/8e27715e-d4b4-11e5-90f7-7a027f89bad3.png">

Can you suggest some things I could do to debug this?

**Comment by wvteijlingen** [comment]
Ouch, this sounds as a hard to debug bug. Smells like something might be going wrong with multithreading maybe? I don't know how your tests are set up, but usually each test starts with a blank slate because of setup/teardown.

Since it only happens when running multiple tests after each other, perhaps as a start you could narrow down the combination of tests in where this happens.

**Comment by kurko** [comment]
Yeah. I spent a good amount of time trying to do that yesterday. Will continue today.

**Comment by kurko** [comment]
**For the record:** I got also another error which I think is unrelated to Spine itself, reason why I didn't post here. It's weird that `NSURLSessionConfiguration.defaultSessionConfiguration()` would blow like this.

I will leave here for the record in case other people see this problem so they can Google:

<img width="993" alt="screen shot 2016-02-17 at 7 24 35 pm" src="https://cloud.githubusercontent.com/assets/70312/13125286/2795f67e-d5ac-11e5-9981-4610483df507.png">

Basically `NSURLSession(configuration: configuration)` gives me a `EXC_BAD_ACCESS(code=EXC_I386_GPFLT)`. It's also random.

**Comment by wvteijlingen** [comment]
I'm closing this as stale. Feel free to reopen if you run into it again :).
